packaging/installer-linux/installer-debian/build.xml

```diff
@@ -25,10 +25,47 @@
                 </fileset>
             </copy>
 
+            <copy toFile="${project.build.outputDirectory}/@{name}/server/default/neo4j">
+                <fileset dir="${project.build.sourceDirectory}/../resources/common">
+                    <include name="default-neo4j"/>
+                </fileset>
+            </copy>
+
             <replace file="${project.build.outputDirectory}/@{name}/server/conf/neo4j-wrapper.conf"
                      token="unsupported.dbms.udc.source=tarball"
                      value="unsupported.dbms.udc.source=debian"/>
 
+            <!-- Remove the files that are unnecessary, or don't make sense on Debian. -->
+            <delete file="${project.build.outputDirectory}/@{name}/server/bin/.keep"/>
+            <delete file="${project.build.outputDirectory}/@{name}/server/plugins/.keep"/>
+            <delete dir="${project.build.outputDirectory}/@{name}/server/bin/tools"/>
```

Comment:
This shouldn't be in the tarball at all. Please remove it at source rather than here.